tests/gtest/avifprogressivetest.cc

```diff
@@ -81,13 +81,13 @@ TEST_F(ProgressiveTest, QualityChange) {
   TestDecode(kImageSize, kImageSize);
 }
 
-// TODO(wtc): Investigate why the following assertion in libaom failed:
+// NOTE: This test requires libaom v3.6.0 or later, otherwise the following
+// assertion in libaom fails:
 //   av1/encoder/mcomp.c:1717: av1_full_pixel_search: Assertion
 //   `ms_params->ms_buffers.ref->stride == ms_params->search_sites->stride'
 //   failed.
-// ms_params->ms_buffers.ref->stride is 832, and
-// ms_params->search_sites->stride is 0.
-TEST_F(ProgressiveTest, DISABLED_DimensionChange) {
+// See https://aomedia.googlesource.com/aom/+/945edd671.
```

Comment:
James: Is this what you suggested?

The output from `git log -1 --oneline` is:

```
$ git log -1 --oneline
945edd671 (HEAD) Fix a bug where an uninitalized search_site is used
```

Reply:
yes, minus the (HEAD) part

Reply:
James: Did you mean the comment should say something like

```
// See the libaom commit
// 945edd671 Fix a bug where an uninitalized search_site is used
```

Reply:
yes, only because the commit hash is stable and a part of the repository itself rather than the review site. A link to aomedia.googlesource.com could work too since even if that fails the commit is still a part of the link.